test/validation/api/classification/odp_classification_basic.c

```diff
@@ -18,7 +18,10 @@ static void test_defaults(uint8_t fill)
 
 	memset(&cos_param, fill, sizeof(cos_param));
 	odp_cls_cos_param_init(&cos_param);
-	CU_ASSERT_EQUAL(cos_param.num_queue, 1);
+
+	CU_ASSERT(cos_param.action == ODP_COS_ACTION_ENQUEUE);
+	CU_ASSERT(cos_param.num_queue == 1);
```

Comment:
This change of the assert macro used for num_queue seems unnecessary. Otherwise looks look.

Reply:
After this, CU_ASSERT_EQUAL() used only for boolean checks. CU_ASSERT() is more commonly used for everything else in validation tests.